components/details_feedback/child_components/comment/details_feedback_comment.react.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Message from '../message/details_feedback_message.react';
+import PropTypes from 'prop-types';
+
+const Comment = props => {
+  const { id, commentLabel, commentWarning, onChange, autoFocus } = props;
+  return (
+    <div className="comment">
+      <label for={id}>{commentLabel}</label>
+      <input onChange={onChange} type="text" id={id} autoFocus={autoFocus} />
+
+      {commentWarning ? <Message message={commentWarning} /> : null}
+    </div>
+  );
+};
+
+Comment.defaultProps = {
+  id: null,
+  commentLabel: null,
+  onChange: null,
+  autoFocus: null
+};
+
+Comment.propTypes = {
```

Comment:
Do we need a proptype for `commentWarning`

Reply:
fixed